dashboard/src/components/DeploymentForm/DeploymentForm.tsx

```diff
@@ -193,7 +197,17 @@ class DeploymentForm extends React.Component<IDeploymentFormProps, IDeploymentFo
             </div>
             <div className="col-8">
               <div>
-                <label htmlFor="releaseName">Name</label>
+                <label htmlFor="hrName">Name</label>
+                <input
+                  id="hrName"
+                  onChange={this.handleHelmReleaseNameChange}
+                  value={this.state.hrName}
+                  required={true}
+                  disabled={hr ? true : false}
+                />
```

Comment:
to confirm that  I've understood it correctly, `hrName` is the name of the helm-crd object and `releaseName`  is the name of the release that tiller creates, right?

Reply:
yes, you understood that correctly, to make it more clear we can change it for `helmCRDRelease` `tillerRelease`. It's longer but easier to understand, isn't it?

Reply:
yeah. that's better.

Reply:
I've done that with #330